static-site/components/list-resources/icon-container.tsx

```diff
@@ -31,10 +24,19 @@ export default function IconContainer({
   iconName,
   text,
 }: {
+  /** (optional) string value for CSS `color` */
   color?: string;
+
+  /** (optional) callback for `onClick` handler */
   handleClick?: () => void;
+
+  /** (optional) string value for CSS `color` when element is hovered over */
   hoverColor?: string;
```

Comment:
Suggestion: remove `(optional)`

```suggestion
  /** string value for CSS `color` */
  color?: string;

  /** callback for `onClick` handler */
  handleClick?: () => void;

  /** string value for CSS `color` when element is hovered over */
  hoverColor?: string;
```

Optional is implied by `?` and I don't think it's necessary to call out unless calling out the default behavior when unset ([example](https://github.com/nextstrain/nextstrain.org/blob/495b6948e22c399b3e0974e57fee6ac718203022/static-site/components/spacers/index.tsx#L20)). Mostly a matter of consistency - it's not used in other parts of the code such as

https://github.com/nextstrain/nextstrain.org/blob/495b6948e22c399b3e0974e57fee6ac718203022/static-site/components/list-resources/use-sort-and-filter.ts#L35-L36